ovh/client.py

```diff
@@ -487,7 +487,7 @@ def raw_call(self, method, path, data=None, need_auth=True):
         }
 
         # include payload
-        if data is not None:
+        if data:
```

Comment:
I fear that this fix won't work because if an API request has its body type to `boolean`, then if a customer want to send `False` as body, it won't be rendered.

I guess we should have a condition more like this:

 ```suggestion
        if data is not None and (not isinstance(data, dict) or data):
```

What do you think about ?